### PR TITLE
Release 2.9.5 beta1

### PR DESCRIPTION
**Pull request by tidy-dev**
## Description

Looking for the PR for the upcoming first beta of the v2.9.5 series? Well, you've just found it, congratulations!

## Release checklist

- [x] Check to see if there are any errors in Sentry that have only occurred since the last production release
- [x] Verify that all feature flags are flipped appropriately
---  No changes - Check run work is still flagged to development.
image

- [x] If there are any new metrics, ensure that central and desktop.github.com have been updated
--- No new metrics

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "2.9.5-beta1": [
+      "[Added] Support PyCharm as an external editor - #13270. Thanks @stevetaggart!",
+      "[Fixed] Custom tooltips - #6384",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Added] Add custom tooltips for quicker and more helpful information - #6384",
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+      "[Fixed] Replace dialog animations with fade when user prefers reduced motion - #12976 #2768",
+      "[Fixed] Fix inactive selected state on lists disappearing - #13115",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] Prevent inactive selected state from disappearing on lists - #13115",
```

**Review by billygriffin — COMMENTED** [review]
A couple minor suggestions, and otherwise LGTM! Thanks!

**Review by billygriffin — APPROVED** [review]
Looks great! 🚀

**Review by Matthelonianxl — COMMENTED** [review]
…

**Pull request merged**